RELEASE.md

```diff
@@ -1,12 +1,17 @@
 # Release 0.17.8
 
 ## Major features and improvements
+
+* Added option to `SparkDataSet` to specify a `schema` load argument that allows for supplying a user-defined schema as opposed to relying on the schema inference of Spark.
```

Comment:
This will now have to go under the `0.18.1` header, but I think that should be solved when fixing the merge conflicts.